message/messaging_service.hh

```diff
@@ -260,6 +260,7 @@ public:
         rack,
         dc,
         all,
+        transitional, // encrypt all outgoing, but do not enforce incoming.
```

Comment:
I don't see support for it in scylla_main

Reply:
Sorry. Managed to fail adding the file.